client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -454,6 +457,39 @@ export class List extends React.Component {
 		);
 	}
 
+	renderPrimaryDomain() {
+		const { domains, translate } = this.props;
+		const primaryDomain = find( domains, 'isPrimary' );
+
+		if ( this.isLoading() || ! primaryDomain ) {
+			return <ListItemPlaceholder />;
+		}
+
+		return [
+			<CompactCard className="list__header-primary-domain" key="primary-domain-header">
+				<div className="list__header-primary-domain-info">
+					{ translate( 'Primary domain' ) }
+					<InfoPopover iconSize={ 18 }>
+						{ translate(
+							'Your primary domain is the address ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Your primary domain is the address visitors will see in their address bar when visiting your blog. All other domains will redirect to the primary domain.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/domains-make-primary-domains-more-prominent)